### PR TITLE
Feature/9037 adjust info screen release 2.11

### PR DESCRIPTION
**Pull request by naveeddotio**
## Description
Changes to info screen for release 2.11

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-9307

## Screenshots
Simulator Screen Shot - Clone 1 of iPhone 12 - 2021-09-21 at 11 07 32
Simulator Screen Shot - Clone 1 of iPhone 12 - 2021-09-21 at 11 07 34

### CODE REVIEW

**Review by InkenTownsend — APPROVED** [review]
UA approved

**Review by marcussc — APPROVED** [review]

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -1595,15 +1595,19 @@ Bei ausgeschalteter Hintergrundaktualisierung müssen Sie die App täglich aufru
 
 "NewVersionFeatures_Info_about_abb_information" = "Änderungen zum Release finden Sie in den App-Informationen unter dem Menüpunkt „Neue Funktionen”.";
 
-/* Version 2.10 */
+/* Version 2.11 */
 
-"NewVersionFeature_210_boosterVaccination_title" = "Information zu Auffrischungsimpfung";
+"NewVersionFeature_211_universalQRCodeScanner_title" = "Universeller QR-Code Scanner";
```
Comment (InkenTownsend) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:1600]:
```suggestion
"NewVersionFeature_211_universalQRCodeScanner_title" = "Universeller QR-Code-Scanner";
```

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 2]
```diff
@@ -1595,15 +1595,19 @@ Bei ausgeschalteter Hintergrundaktualisierung müssen Sie die App täglich aufru
...
 
-"NewVersionFeature_210_boosterVaccination_description" = "Die App kann Ihnen nun eine Mitteilung anzeigen, die Sie über eine eventuell anstehende Auffrischungsimpfung informiert.";
+"NewVersionFeature_211_universalQRCodeScanner_description" = "Sie können nun unabhängig davon, von wo ein QR-Code Scanner in der CWA geöffnet wird, alle von der CWA unterstützen Dokumente scannen:\n\t\U2022 PCR-Tests\n\t\U2022 Schnelltests\n\t\U2022 Event Check-ins\n\t\U2022 Zertifikate";
```
Comment (InkenTownsend) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:1602]:
```suggestion
"NewVersionFeature_211_universalQRCodeScanner_description" = "Sie können nun unabhängig davon, von wo ein QR-Code-Scanner in der CWA geöffnet wird, alle von der CWA unterstützen Dokumente scannen:\n\t\U2022 PCR-Tests\n\t\U2022 Schnelltests\n\t\U2022 Event Check-ins\n\t\U2022 Zertifikate";
```

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 3]
```diff
@@ -1595,15 +1595,19 @@ Bei ausgeschalteter Hintergrundaktualisierung müssen Sie die App täglich aufru
...
 
-"NewVersionFeature_210_printVersionVaccinationCertificate_title" = "Druckversion von Impfzertifikaten";
+"NewVersionFeature_211_qrCodeImport_title" = "QR-Code importieren";
 
-"NewVersionFeature_210_printVersionVaccinationCertificate_description" = "Sie können nun eine Druckversion Ihrer in Deutschland ausgestellten Zertifikate anzeigen, die es Ihnen ermöglicht, die Zertifikate als PDF-Dokument zu speichern oder auszudrucken.";
+"NewVersionFeature_211_qrCodeImport_description" = "Sie können nun auch QR-Codes von Fotos oder PDF-Dateien über den QR-Code Scanner importieren.";
```
Comment (InkenTownsend) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:1606]:
```suggestion
"NewVersionFeature_211_qrCodeImport_description" = "Sie können nun auch QR-Codes von Fotos oder PDF-Dateien über den QR-Code-Scanner importieren.";
```

**Comment by InkenTownsend** [comment]
@naveeddotio I realized that we spelled QR-Code-Scanner with two - before and I adjusted it before the merge. Could you please take over my changes and check that I havent missed it anywhere? Thank you very much and sorry!

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings [hunk 4]
```diff
@@ -1595,15 +1595,19 @@ Bei ausgeschalteter Hintergrundaktualisierung müssen Sie die App täglich aufru
...
+/* Version 2.11 */
 
-"NewVersionFeature_210_boosterVaccination_title" = "Information zu Auffrischungsimpfung";
+"NewVersionFeature_211_universalQRCodeScanner_title" = "Universeller QR-Code-Scanner";
 
-"NewVersionFeature_210_boosterVaccination_description" = "Die App kann Ihnen nun eine Mitteilung anzeigen, die Sie über eine eventuell anstehende Auffrischungsimpfung informiert.";
+"NewVersionFeature_211_universalQRCodeScanner_description" = "Sie können nun unabhängig davon, von wo ein QR-Code-Scanner in der CWA geöffnet wird, alle von der CWA unterstützen Dokumente scannen:\n\t\U2022 PCR-Tests\n\t\U2022 Schnelltests\n\t\U2022 Event Check-ins\n\t\U2022 Zertifikate";
 
-"NewVersionFeature_210_printVersionVaccinationCertificate_title" = "Druckversion von Impfzertifikaten";
+"NewVersionFeature_211_qrCodeImport_title" = "QR-Code importieren";
```
Comment (ChristinaTsigganos) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:1604]: @naveeddotio As the feature "QR-Code importieren" will not be delivered with 2.11 for iOS, can you please remove the according bullet point from the text? Figma and JIRA are already adjusted.

**Review by ChristinaTsigganos — CHANGES_REQUESTED** [review]
@naveeddotio As the feature "QR-Code importieren" will not be delivered with 2.11 for iOS, can you please remove the according bullet point from the text? Figma and JIRA are already adjusted.

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings (hunk 4 above)
Comment (naveeddotio) [reply src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings:1604]: @ChristinaTsigganos Yes, I am aware about this. It will be adjusted.

**Review by ChristinaTsigganos — APPROVED** [review]
UA approved

**Pull request merged**